__tests__/e2e/scripts/test.sh

```diff
@@ -23,5 +23,13 @@ echo "✅ Setup completed successfully. Running e2e tests..."
 #
 # run e2e tests journey
 ##################################################
+set +e
```

Comment:
Enable error to trap the errorlevel to manipulate the junit report

This could be moved to the reporter junit but while it's not there this is the approach to run this in the CI